tests/acceptance/features/coreApiShareManagementBasicToShares/createShareToSharesFolder.feature

```diff
@@ -643,8 +643,8 @@ Feature: sharing
       | permissionsRole | File Editor   |
     When user "Alice" shares file "textfile0.txt" with group "grp1" using the sharing API
     Then the HTTP status code should be "<http-status>"
-    And the OCS status code should be "403"
-    And the OCS status message should be "Path already shared with this group"
+    And the OCS status code should be "104"
```

Comment:
We have a test for sharing NG, should we remove the whole scenario or let it be there with adjusted expectations?
@saw-jan @phil-davis

Reply:
I think that it is good to put what happens in "real life" and have the test passing.

They will get deleted later, whenever OCS is actually removed.